### PR TITLE
Extra services info

### PR DESCRIPTION
**Pull request by qbalukom**
**Description**
addded "Service Type" and "External IPs" to service list and details

### CODE REVIEW

File: core-ui/src/components/Predefined/List/Services.list.js [hunk 1]
```diff
@@ -1,29 +1,41 @@
...
-      return '';
+      return `${port.port}:${port.targetPort}/${port.protocol}`;
     }
   };
 
+  const getPorts = service =>
+    intersperse(service.spec.ports?.map(getPortString) || [], ', ');
+
+  const getExternalIPs = service =>
+    intersperse(service.spec.externalIPs || [], ', ');
```
Comment (Wawrzyn321) [inline core-ui/src/components/Predefined/List/Services.list.js:21]: Wouldn't using the `Array<string>::join(', ')` be just enough? I know one evaluates to an array (of strings in this case), the `join` returns a single string, but for rendering on UI the result will be the same.

File: core-ui/src/components/Predefined/Details/Service/Service.details.js [hunk 2]
```diff
@@ -90,6 +96,16 @@ export const ServicesDetails = ({ DefaultRenderer, ...otherParams }) => {
...
       ),
     },
+    {
+      header: t('services.external-ips'),
+      value: service => (
+        <ul>
+          {service.spec.externalIPs?.map(ip => (
+            <li key={ip}>{ip}</li>
+          ))}
+        </ul>
```
Comment (Wawrzyn321) [inline core-ui/src/components/Predefined/Details/Service/Service.details.js:106]: Nitpicking time! When the value is empty, we use '-' as a placeholder, so please use it in `externalIPs` field on both details and list.

Comment (Wawrzyn321) [reply core-ui/src/components/Predefined/Details/Service/Service.details.js:106]: Example - when there are no labels.

File: core-ui/src/components/Predefined/List/Services.list.js (hunk 1 above)
Comment (qbalukom) [reply core-ui/src/components/Predefined/List/Services.list.js:21]: technically it came from a few ideas along the way... didn't feel like getting rid of intersperse - could be useful in the future :)

**Review by Wawrzyn321 — APPROVED** [review]

**Review by Wawrzyn321 — APPROVED** [review]

**Review by NHingerl — APPROVED** [review]

**Pull request merged**